jquery.ajax.fake.js

```diff
@@ -2,83 +2,177 @@
  * jquery.ajax.fake.js
  * http://anasnakawa.github.com/jquery.ajax.fake
  * license: MIT (http://opensource.org/licenses/mit-license.php)
+ * author : Anas Nakawa anas.nakawa@gmail.com @anasnakawa
  * ------------------------------
+ *
+ * Custom version for Rakuten
```

Comment:
@rogeriotaques  If it is a Custom version, why you're doing a pull request to the repo ?